Kitodo-DataManagement/src/main/java/org/kitodo/data/database/beans/User.java

```diff
@@ -321,7 +322,7 @@ public User selfDestruct() {
     // Here will be methods which should be in UserService but are used by jsp
     // files
 
-    private static final Logger logger = Logger.getLogger(MySQLHelper.class);
+    private static final Logger logger = LogManager.getLogger(MySQLHelper.class);
```

Comment:
Can you please correct the used class name from `MySQLHelper.class` to `User.class` as the class name is `User`?